test/integration/scheduler/scheduler_test.go

```diff
@@ -615,13 +615,14 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			util.ExpectClusterQueueToBeDeleted(ctx, k8sClient, cq, true)
 			util.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, onDemandFlavor, true)
 		})
-		ginkgo.It("Should re-enqueue by the update event of ClusterQueue", func() {
+		ginkgo.FIt("Should re-enqueue by the update event of ClusterQueue", func() {
```

Comment:
```suggestion
		ginkgo.It("Should re-enqueue by the update event of ClusterQueue", func() {
```

Debug code still remain :)